livebooks/messages_source_and_sink/messages_source_and_sink.livemd

```diff
@@ -18,7 +18,7 @@ defmodule MessageSource do
   require Membrane.Logger
 
   def_output_pad(:output,
-    mode: :push,
+    flow_control: :push,
     accepted_format: _any
   )
 
```

Comment:
Remove the brackets from the pad definitions

Reply:
I did it, but Livebook does some kind of auto-formatting and adds these brackets every time you just open the file in Livebook.